packages/components/src/toggle-multiple-group-control/component.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import type { ForwardedRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	contextConnect,
+	ContextSystemProvider,
+	WordPressComponentProps,
+} from '../ui/context';
+import type { ToggleMultipleGroupControlProps } from './types';
+import { ToggleGroupControl } from '../toggle-group-control';
+import { useContextSystem } from '../ui/context/use-context-system';
+
+const contextProviderValue = {
+	ToggleGroupControlOptionBase: {
+		isMultiple: true,
```

Comment:
Instead of using the context system, could we instead pass the `isMultiple` prop directly to the `ToggleGroupControlOptionIcon` inside `ToggleMultipleGroupControlOptionIcon`?

Maybe it could be a bit easier to discover for a developer?  WDYT ? (not a strong opinion, though)